geode-docs/tools_modules/gfsh/command-pages/connect.html.md.erb

```diff
@@ -193,8 +193,8 @@ http://myLocatorHost.example.com:8080/gemfire/v1
 "Locator could not find a JMX Manager";
 "jmx password must be specified.";
 "Could not connect to : {0}. {1}";
-"Could not find a GemFire jmx-manager service running at {0}.";
+"Could not find a Geode jmx-manager service running at {0}.";
```

Comment:
I have changed lines 196 & 199 to show how these strings really look (they are defined in CliStrings.java)

Reply:
@alb3rtobr @moleske +1 good catch.